src/plugins/data/common/index_patterns/index_patterns/ensure_default_index_pattern.ts

```diff
@@ -36,13 +36,18 @@ export type EnsureDefaultIndexPattern = () => Promise<unknown | void> | undefine
 
 export const createEnsureDefaultIndexPattern = (
   uiSettings: UiSettingsCommon,
-  onRedirectNoIndexPattern: () => Promise<unknown> | void
+  onRedirectNoIndexPattern: () => Promise<unknown> | void,
+  isCurrentWorkspaceOwner?: boolean
```

Comment:
Can we rename it to like `isAbleToUpdateUISettings`?

Reply:
updated.